src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala

```diff
@@ -62,8 +64,8 @@ object DDM extends DebugEnhancedLogging {
        { /* instructions for reuse not specified as such in EMD */ }
        { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue.trim }</dc:creator>) }
        { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
-       { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
-       { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+       { if (dateCreated.nonEmpty)  <ddm:created>{ dateCreated.toSeq.head }</ddm:created> }
+       { dateAvailable.map(date =>  <ddm:available>{ date }</ddm:available>) }
```

Comment:
Does this give the result you want? I'd expect it returns something with `Node` as prefix.

Reply:
the `dateCreated` and `dateAvailable` no longer contain Nodes, they have been parsed already. I hope the changed variable name would hint at that

Reply:
Overlooked the change from node to date.